hbp_knowledge/receptors/buckingham2009.bel

```diff
@@ -1559,23 +1685,25 @@ in the locus ceruleus of brains from patients with
 AD compared with brains from subjects without AD,
 whereas no change was observed for adrenergic brainstem
 nuclei (Strong et al., 1991)."
-
+SET Confidence = "High"
 SET MeSHAnatomy="Basal Ganglia"
 
 path(MESH:"Alzheimer Disease") -| a(MESH:"Cholinergic Neurons")
 path(MESH:"Alzheimer Disease") -| act(p(HGNC:CHAT))
 
 UNSET MeSHAnatomy
-
+UNSET Confidence
 SET Evidence="Selective cell loss in the locus ceruleus has
 also been confirmed in a double-mutant mouse model
 ([APPswe, PS1dE9]85Dbo/J; PrP promoter) transgenically
 expressing the Swedish mutant form of the APP
 and the mutant human APP/PS-1, suggesting that these
 mutations can contribute to neurodegeneration (O’Neil
 et al., 2007)."
-
+SET Confidence = "Low"
 #pass
+#not sure how to represnt "Chr21:27269939 G>T, Chr21:27269938 A>C". This is what they mean by Swedish mutant form of APP.
```

Comment:
you can put the proteins as variants.
So the mutation on the level of aminoacids would be K595 -> N and M596 -> L
```suggestion
p(HGNC:MAPT, var("p.Lys595Asp")) -> path(HP:Neurodegeneration)
p(HGNC:MAPT, var("p.Met596Leu")) -> path(HP:Neurodegeneration)
```

(you can also represent a mutation on the level of DNA, but you would need the position of the mutated base in the gene and not on the chromosome. you can look this up)